public/app/features/inspector/QueryInspector.tsx

```diff
@@ -206,7 +206,7 @@ export class QueryInspector extends PureComponent<Props, State> {
 
   getNrOfOpenNodes = () => {
     if (this.state.allNodesExpanded === null) {
-      return 3; // 3 is default, ie when state is null
+      return 4; // 4 is default, ie when state is null
```

Comment:
This change was required to get e2e tests working more easily -- essentialy it now starts with the response open one level deeper:

![image](https://user-images.githubusercontent.com/705951/112941294-310f6300-90e3-11eb-946d-37689903e685.png)


Before it was:
![image](https://user-images.githubusercontent.com/705951/112941389-5e5c1100-90e3-11eb-84c4-1f4cb7d26e9a.png)